frontends/open-discussions/webpack.config.js

```diff
@@ -62,10 +62,11 @@ const getWebpackConfig = mode => {
     },
     plugins: [
       new BundleTracker({ filename: STATS_FILEPATH }),
+      new webpack.ProvidePlugin({
+        process: "process/browser",
+      }),
       new webpack.DefinePlugin({
-        "process.env": {
-          env: { NODE_ENV: JSON.stringify(mode) },
-        }
+        NODE_ENV: JSON.stringify(process.env.NODE_ENV)
       }),
```

Comment:
Webpack 4 polyfilled node modules such as `process` for the browser. Webpack 5 does not.

This polyfills process since some of our dependencies use it. *Other polyfilles, like `assert`, are proided by Webpack's `fallback` option. That is good for things that dependencies import, but dependencies expect `process` to be available on the global object, hence use of `ProvidePlugin`.*

I couldn't figure out how to polyfill process AND set the browser's `process.env.NODE_ENV` based the node NODE_ENV. So `NODE_ENV` now gets set on the window object via DefinePlugin.